src/main/java/com/bakdata/common_kafka_streams/KafkaStreamsApplication.java

```diff
@@ -257,6 +261,13 @@ protected void runCleanUp() {
             }
         }
         this.streams.cleanUp();
+
+        try {
+            this.resetSchemaRegistry();
```

Comment:
only do that if deleteOutput is set. Therefore, I suggest that you add this method call to deleteTopic()